messages/de.json

```
@@ -457,6 +515,7 @@
 	"rollCollError": "Gremienmitglied nicht gefunden",
 	"rollCollSuccess": "Anwesenheitsfeststellung abgeschlossen",
 	"save": "Speichern",
```

Biome (2.4.14)

[error] The key save was already declared.

(lint/suspicious/noDuplicateObjectKeys)